### DISCUSSION TITLE
How to create new account and password

### DISCUSSION THREAD

**Discussion by danieleyny** [microrealestate/microrealestate#79] [Q&A]
How do we create a new username and password? When I open the system I automatically get the demo mode username and password filled out and when I log in it brings me to the system with prefilled data. But I do not see a way to create a new username and password to start with a blank system and add my own data. Thank you for the help.

**Comment by camelaissani** [comment]
Choose the option `empty data` when the `mre` prompt `Do you want the database to be populated with`

As you have already run the `mre` command you will have to delete first the `.env` file at the root of the project directory.
